src/Sulu/Bundle/WebsiteBundle/Cache/CacheClearer.php

```diff
@@ -53,20 +53,27 @@ class CacheClearer implements CacheClearerInterface
      */
     private $eventDispatcher;
 
+    /**
+     * @var bool
+     */
+    private $tagsEnabled;
+
     public function __construct(
         Filesystem $filesystem,
         $kernelEnvironment,
         RequestStack $requestStack,
         EventDispatcherInterface $eventDispatcher,
         string $varDir,
-        ?CacheManager $cacheManager
+        ?CacheManager $cacheManager,
+        bool $tagsEnabled
```

Comment:
add a default value here to avoid a bc break:

```suggestion
        bool $tagsEnabled = true
```